airbyte-integrations/connectors/source-greenhouse/source_greenhouse/manifest.yaml

```diff
@@ -3204,15 +3225,12 @@ definitions:
     retriever:
       $ref: "#/definitions/retriever"
       requester: "#/definitions/requester"
-      stream_slicer:
-        request_cursor_field: "submitted_after"
-        cursor_field: "submitted_at"
-        class_name: source_greenhouse.components.GreenHouseSlicer
     incremental_sync:
-      type: CustomIncrementalSync
-      class_name: source_greenhouse.components.GreenHouseSlicer
-      request_cursor_field: "submitted_after"
+      $ref: "#/definitions/base_incremental_stream/incremental_sync"
       cursor_field: "submitted_at"
+      start_time_option:
+        $ref: "#/definitions/base_incremental_stream/incremental_sync/start_time_option"
```

Comment:
very small nit, but rather than reference the existing `start_time_option` and overriding one field, it might be a little cleaner to just redefine `start_time_option` since its such a simple object. It's just a little bit more readable:
```
start_time_option
  type: RequestOption
  inject_into: request_parameter
  field_name: "submitted_after"
```